### PR TITLE
Added minified file into package

### PR DESCRIPTION
**Pull request by ivanfilhoz**
Solving issue #16, by including a task to gruntfile which automatically updates minified version.

### CODE REVIEW

File: Gruntfile.js [hunk 1]
```diff
@@ -24,6 +24,13 @@ module.exports = function (grunt) {
                 config: '.jscsrc'
             },
             src: '<%= config.lintFiles %>'
+        },
+        uglify: {
+            build: {
+                files: {
+                    'angular-bind-html-compile.min.js': 'angular-bind-html-compile.js',
```
Comment (wytrych) [inline Gruntfile.js]: Please remove trailing comma

**Comment by wytrych** [comment]
Nice work, thanks!

**Comment by wytrych** [comment]
Looks like you need to remove the `min` file from `config.lintFiles`

**Comment by ivanfilhoz** [comment]
- Removed trailing comma
- Disabled jshint for minified version

All ok now!

File: Gruntfile.js [hunk 2]
```diff
@@ -8,7 +8,7 @@ module.exports = function (grunt) {
         // Configurable paths
         config: {
             lintFiles: [
-                '**/*.js'
+                'angular-bind-html-compile.js'
```
Comment (wytrych) [inline Gruntfile.js]: Could you change this to  

```
[
    '**/*.js',
    '!*.min.js'
]
```

So we lint the Gruntfile too and any future potential files?

**Comment by wytrych** [comment]
You probably don't have `[user]`  and `[github]` set with the correct username and email, as github thinks you are unauthorised

**Comment by wytrych** [comment]
I have this in my `~/.gitconfig`:

```
[user]
    email = marcin@incuna.com
[github]
    user = wytrych
```

**Comment by ivanfilhoz** [comment]
Done.

**Comment by wytrych** [comment]
Thank you!

**Comment by wytrych** [comment]
Released under `1.2.0`

**Pull request merged**